packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -744,7 +744,19 @@ export class SingleThreadedWallet
 
   async channelFinalized(arg: ChannelFinalizedArg): Promise<void> {
     const response = WalletResponse.initialize();
+
     await this.store.setFinalizedChallengeStatus(arg.channelId, arg.blockNumber);
+    await this.knex.transaction(async tx => {
+      const {objectiveId} = await this.store.ensureObjective(
+        {
+          type: 'DefundChannel',
+          participants: [],
```

Comment:
Why is `participants` a property of this objective if it's empty?

Reply:
Since `wallet-core` defines `participants` as a property on the `objective` and `ensureObjective` takes a `Objective` from `wallet-core`.

Sidenote: I'd personally prefer having the `server-wallet` stop using `wallet-core` for the `objective` structure.

```
type _Objective<Name, Data> = {
  participants?: Participant[];
  type: Name;
  data: Data;
};
```